cmd/podman/utils/utils.go

```diff
@@ -104,6 +106,62 @@ func PrintNetworkPruneResults(networkPruneReport []*entities.NetworkPruneReport,
 	return errs.PrintErrors()
 }
 
+func PrintSystemCheckResults(report *entities.SystemCheckReport) error {
```

Comment:
i think? there is precedent for output related functions to live in cmd/podman/foo alongside the command itself?  I think we have a bunch of PrintJSON and so forths.

Reply:
Ah, for the ones that aren't called from multiple packages, that does appear to be the pattern.  Moved it.